content/Registration and Authentication in your Django app with dj-rest-auth part 2 - Bruno Michetti.md

```diff
@@ -0,0 +1,287 @@
+## Registration and Authentication in your Django app with dj-rest-auth (part 2)
+
+In the [first part of this blog post](link-to-part-1) I've presented the dj-rest-auth package that allows us to handle registration and authentication through REST API endpoints with a minimal configuration.
+I've also talked about different customizations that you can take advantage of.
+When I started to use this package I had to face some problems. The roots of those problems were partly because I was learning Django and partly because dj-rest-auth has many features and possibilities for each feature that the first time is used it can lead to some confusion. So the main goal of the second part is to show you more examples and give you some tips.  
```

Comment:
```suggestion
When I started to use this package I faced some problems because I was learning Django and dj-rest-auth offers many features and possibilities for each feature. This is why the first users may experience some confusion. I will be sharing more examples and give you some tips.  
```